src/sentry/static/sentry/app/components/events/interfaces/stacktraceContent.tsx

```diff
@@ -141,6 +157,7 @@ export default class StacktraceContent extends React.Component {
             onAddressToggle={this.handleToggleAddresses}
             image={image}
             maxLengthOfRelativeAddress={maxLengthOfAllRelativeAddresses}
+            registers={{}} //TODO: Fix registers
```

Comment:
Why not change FrameLine Prop type in this pull request? It looks like it only needs to be made optional.

Reply:
@markstory I would have to go down into `FrameContext` and propagate changes there adding more logic like `&& registers`. I also wonder if we should actually be passing `frame.registers` here as `registers` as a prop.